src/api/exporters/ezsLocals/JSONLDObject.js

```diff
@@ -61,15 +61,28 @@ function mergeSimpleField(output, field, data) {
     };
 }
 
-function getUri(uri) {
-    if (uri.indexOf('http://') !== 0 &&
-        uri.indexOf('https://') !== 0) {
-        return path.normalize(this.getParam('hostname', 'http://lod.istex.fr/').concat(uri));
+function removeNumberInstance(uri) {
+    const reg = new RegExp('(\\-\\d+)(\\.[a-z]+)+');
+    const match = reg.exec(uri);
+
+    if (match !== null) {
+        return uri.replace(match[1], '');
     }
 
     return uri;
 }
 
+function getUri(uri) {
+    const u = removeNumberInstance(uri);
+
+    if (u.indexOf('http://') !== 0 &&
+        u.indexOf('https://') !== 0) {
+        return path.normalize(this.getParam('hostname', 'http://lod.istex.fr/').concat(u));
```

Comment:
now, "this" is undefined, before "this" was the ezs scope.  this.getParam('hostname') is used to fetch ezs statement parameter (https://github.com/touv/node-ezs#scope)

of course, we should use "host" from defaut.js, and to get it, one way is to use ezs parameter like this https://github.com/Inist-CNRS/lodex/blob/master/src/api/exporters/exportJsonld.js#L14